client/my-sites/controller.js

```diff
@@ -60,10 +60,11 @@ import {
 } from 'calypso/my-sites/domains/paths';
 import {
 	emailManagement,
-	emailManagementForwarding,
 	emailManagementAddGSuiteUsers,
-	emailManagementNewGSuiteAccount,
+	emailManagementForwarding,
+	emailManagementManageTitanMailboxes,
```

Comment:
Nit: This import should be below the next line to keep alphabetical order.

Reply:
Good catch.